tests/integration/security/testdata/beta-mtls-partial-automtls.yaml

```diff
@@ -35,3 +35,17 @@ spec:
     # 8090 is the targetPort for service http(80)
     8090:
       mode: STRICT
+---
+apiVersion: "security.istio.io/v1beta1"
+kind: "PeerAuthentication"
+metadata:
+  name: "headless"
+  annotations:
+    test-suite: "beta-mtls-partial-automtls"
+spec:
+  selector:
+    matchLabels:
+      app: headless
+  mtls:
```

Comment:
same you should only need to change go code.

Reply:
I can add a new test case